pkg/adaptor/mongodb/mongodb.go

```diff
@@ -359,35 +359,36 @@ func (m *MongoDB) catData() error {
 		} else if match := m.collectionMatch.MatchString(collection); !match {
 			continue
 		}
-		query := bson.M{}
-		// lastID := ""
+		canReissueQuery := m.requeryable(collection)
```

Comment:
yea, I wouldn't expect people to be messing with the primary `_id` index much to begin with much less *while* trying to perform a data transfer .